draft-ietf-tls-esni.md

```diff
@@ -157,18 +158,17 @@ server.
 
 ## SNI Encryption
 
-The protocol designed in this document is quite straightforward.
+The protocol designed in this document is straightforward.
```

Comment:
Maybe avoid making untestable claims:
```suggestion
```

More so this set of changes makes the design the very opposite of straightforward.